src/cms/migrations/0015_historicalpage.py

```diff
@@ -20,6 +20,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.IntegerField(auto_created=True, blank=True, db_index=True, verbose_name='ID')),
                 ('support_copy_paste', models.BooleanField(default=True, help_text='Turn this on if copy-pasting content from a word processor, or using the toolbar to format text. It tells Janeway to clear out formatting that does not play nice. Turn it off and leave it off if anyone has added custom HTML or CSS using the code view, since it might remove custom code.')),
+                ('template', models.CharField(blank=True, help_text='The custom template to use instead of the content field.', max_length=100)),
```

Comment:
This migration existed in 1.5.1. If we add a field here, existing installations won't have it installed (the migration has already run) and will see the reported error. I think we need a new migration to add the field.

Altering past migrations is not something we can do for schema altering migrations

Reply:
Apologies, I thought it was new to 1.5.2, but yes you are right. I'll create a new migration.